hack/scripts/sync-prow.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
@timwangmusic FTR this script comes from https://github.com/kubernetes-sigs/prow/pull/74#issuecomment-1907347751. Typically when code is copied from somewhere, you should cite the source to include the additional context of how/why the script came to be. I don't expect you to cite the URL in the code directly, but having it in the PR description would have been helpful for reviewers (for example, to compare against the previous script that you already used to import commits into this repo, which had a different merge direction than this one).

Reply:
Added the reference in the PR description, thanks for pointing this out.